R/compile_tmb.R

```diff
@@ -0,0 +1,40 @@
+# wrapper around TMB::compile which silences annoying C++ warning messages
+#' @title Compile TMB model without C++ Warnings
+#' @description Wrapper around \code{\link[TMB]{compile}} which optionally 
+#' redirects C++ compiler output to a logfile, avoiding your R console, since 
+#' it can be quite verbose. Note that \code{logfile} argument default is not
+#' valid for Windows.
```

Comment:
Up to you but you but you can get away without specifying this explicitly. Can do
```
#' This is the title
#'
#' This is my description whihc might be
#' multiple lines long
#'
#' @param ...
```